src/rez/package_cache.py

```diff
@@ -564,6 +642,15 @@ def run_daemon(self):
             if pid > 0:
                 sys.exit(0)
 
+        self.run_caching_operation(wait_for_copying=False)
+
+    def run_caching_operation(self, wait_for_copying=False):
```

Comment:
I feel like this method should be private unless there is a god reason not to. I also think `wait_for_copying` should be True by default.

Reply:
I'll make it private, but I'd rather not change the default beahvior of the caching from async to sync here.

Reply:
Sorry, this one can be true, you're right.